### ISSUE TITLE
RuntimeError: No Qt binding was found, got: No module named 'PyQt5.QtCore

### ISSUE THREAD

**Issue by braincharter** [pyvista/pyvistaqt#235]
Hello! I just did a fresh install on windows (anaconda, python 3.8), and got this issue. Do I have to install PyQt5? I doesn't seem to solve the issue..

**Comment by larsoner** [comment]
Yes you need to install a suitable Qt binding. Nowadays either PyQt6 or PySide6 should work. Feel free to reopen if you have issues with those!
